src/usfs/santee/processing_kernels.R

```diff
@@ -539,64 +539,92 @@ process_1_VERSIONLESS005 <- function(network, domain, prodname_ms, site_code, co
                     p = prodname_ms,
                     s = site_code)
     
+    # WS79 AND WS80 are miss labeled. Check this if the product is updated
```

Comment:
better to raise a warning/error when the product update occurs. this is possible under non-gdrive versionless product handling, where we check the modification time of the remote file. ideally our versionless files on gdrive would also be checked for last modification time, so that they get updated locally when we change them.